RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -82,18 +82,22 @@ public class PaywallViewController: UIViewController {
                                introEligibility: nil,
                                purchaseHandler: nil)
             .onPurchaseCompleted { [weak self] transaction, customerInfo in
-                guard let self = self else { return }
+                guard let self else { return }
                 self.delegate?.paywallViewController?(self, didFinishPurchasingWith: customerInfo)
                 self.delegate?.paywallViewController?(self,
                                                       didFinishPurchasingWith: customerInfo,
                                                       transaction: transaction)
             }
+            .onPurchaseCancelled { [weak self] in
```

Comment:
FFTI. Should we add this modifier to paywall tester, even if it's just to log a message, so people looking at it for examples know it exists?

Reply:
👍🏻